extensions/functions_arithmetic.yaml

```diff
@@ -745,6 +745,118 @@ scalar_functions:
           overflow:
             values: [ SILENT, SATURATE, ERROR ]
         return: i64
+  -
+    name: "bitwise_not"
```

Comment:
Anybody have strong feeling on these names?

I'd be inclined to use AND, OR, XOR and NOT.

Reply:
> I'd be inclined to use AND, OR, XOR and NOT.

I note there are some boolean functions with these names in `functions_boolean.yaml`. Is it better to keep "bitwise" prefixed here to avoid possible ambiguity? I am new to substrait, so not sure whether there is any conflict issue. Thanks!

Reply:
Presto and Arrow use `bitwise_xyz` (same as here).  SQL server and postgres use the operators `|`, `&`, `^` (XOR, SQL server), `#` (XOR postgres).  Trino has `BIT_AND`, `BIT_OR`, and `XOR`.

I'm slightly in favor of the proposed names as it distinguishes between the existing AND (e.g. I would expect logical AND to have `5 AND 7 => 1`)

Reply:
Sounds good. Just wanted to make sure we felt good about the names.